README.md

```diff
@@ -1,25 +1,42 @@
 # signex
```

Comment:
It would be good to explain how signature string is composed and how headers are joined.